components/generic/SmartLink.vue

```diff
@@ -1,14 +1,15 @@
 <template>
   <b-link
     v-if="useRouterLink"
-    :to="destination"
+    :to="path"
+    :exact-active-class="exactActiveClass"
```

Comment:
Where or when is this used? Isn't it always null?

Reply:
This is used for active state in the header navigation. If you look at Pagenavigation.vue, you can see we are passing `font-weight-bold` to it

Reply:
I am wondering if it is necessary, as nuxt links get an active/exact active class already. You can use that for styling. Eg, if you go to /mirjams-test, and inspect that link in the navigation, you can see the class "nuxt-link-active".